bodhi/server/util.py

```diff
@@ -789,19 +789,21 @@ def call_api(api_url, service_name, error_key=None, method='GET', data={}):
     :param data: a dict of data will be sent to the API server.
     :return: dictionary of the returned JSON or a RuntimeError
     """
+    session = requests.Session()
```

Comment:
This will have a session (which contain the connection pools) created each time the function is called. This will need to be moved outside the function into the module-level so it's created once at import-time.